tests/terraform/graph/checks_infra/attribute_solvers/is_true_solver/TrueValue.yaml

```diff
@@ -0,0 +1,11 @@
+metadata:
+  id: "BooleanString"
+scope:
+  provider: "Azure"
+definition:
+  cond_type: "attribute"
+  resource_types:
+    - "azurerm_storage_account"
+  attribute: "allow_blob_public_access"
+  operator: "is_true"
```

Comment:
I suggest creating a test case where the attribute is actually `False` and we are trying to check for `is_true`